test/wdio/import-aliasing/cmp.test.tsx

```diff
@@ -41,7 +45,8 @@ describe('import aliasing', function () {
   });
 
   it('should link up to the surrounding form', async () => {
-    const formEl = app.querySelector('form');
+    await $('import-aliasing').waitForExist();
+    const formEl = document.querySelector('form');
     expect(new FormData(formEl).get('test-input')).toBe('my default value');
```

Comment:
Similar to my problem above, I didn't know how to get the form element to create the FormData here. Happy to hear suggestions